ligo/src/types.mligo

```diff
@@ -102,11 +104,9 @@ type proposal =
   // has actually paid
   ; proposer_fixed_fee_in_token : nat
 
-  ; voters : (address * nat) list
+  ; voters : (address * (nat * vote_type)) list
```

Comment:
one more thing: we should put this in its own type.

Firstly, because we have to use this signature in one place and that would be better.
Secondly, because we should annotate all the types in storage (especially this one whose purpose is to be read from).

So I suggest adding:
```ocaml
type voter =
  { voter_address : address
  ; vote_amount : nat
  ; vote_type : vote_type
  }
```
and just use a `voter list` here.